x-pack/plugins/lens/public/xy_visualization/xy_config_panel/reference_line_panel.tsx

```diff
@@ -27,8 +27,10 @@ export const ReferenceLinePanel = (
   props: VisualizationDimensionEditorProps<State> & {
     formatFactory: FormatFactory;
     paletteService: PaletteRegistry;
+    layer: XYReferenceLineLayerConfig;
   }
 ) => {
+  const layer = props.layer;
```

Comment:
This change makes the UI build on top of the upstream state instead of the local state, that means changes are only reflected in the config UI after the debounce time:
![Kapture 2022-03-09 at 12 47 59](https://user-images.githubusercontent.com/1508364/157436111-8439637e-9005-4157-8bbe-0ab745f6502b.gif)